### ISSUE TITLE
config模块的初始化

### ISSUE THREAD

**Issue by dxwy2001** [beego/beedoc#912]
InitGlobalInstance主动初始化的时候，由于在包的init的函数里面，会直接用ini实现优先进行初始化？

**Comment by flycash** [comment]
是的。你可以再次调用来初始化它。总体而言这只是一个辅助手段，并不是说你一定要使用这个global instance，即便你要使用，你也可以自己手动初始化一下，指定路径
